scheduler/scheduler.yml

```diff
@@ -177,35 +183,12 @@
   roles:
     - k8s_csi_powerscale_plugin
 
-- name: Apply slurm control node config
-  hosts: slurm_control_node
-  gather_facts: false
-  roles:
-    - slurm_manager
-
-- name: Configure Slurm worker nodes
-  hosts: slurm_node, login
-  gather_facts: false
-  roles:
-    - slurm_workers
-
-- name: Start slurm services on slurm control node
-  hosts: slurm_control_node
-  gather_facts: false
-  roles:
-    - slurm_start_services
-
```

Comment:
If we are deleting this roles invocation, roles also should be deleted in same PR

Reply:
am yet to do the analysis - will take it up next task - have told this earlier also